### ISSUE TITLE
Parameter 'pdf.formatter' of the org.dita.pdf2 plugin.xml has no values

### ISSUE THREAD

**Issue by xephon2** [dita-ot/dita-ot#2213]
The [plugin.xml](https://github.com/dita-ot/dita-ot/blob/develop/src/main/plugins/org.dita.pdf2/plugin.xml#L35) of the org.dita.pdf2 plugin does not comply with the `plugin.rnc`. The param `pdf.formatter` (for setting the XSL processor) does not have at least one `<val>` child.

```xml
<param name="pdf.formatter" desc="Specifies the XSL processor." type="enum"/>
```

When I try to set the parameter via Oxygen, Oxygen allows me to set the parameter to either "Default" or "xep".

**Comment by xephon2** [comment]
BTW: I validated all `plugin.xml` files of all DITA-OT 2.2.2 plugins and this is the only issue I have found.

**Comment by jelovirt** [comment]
This was fixed in a32d667cc22a73e8a54294061ca639f6f80f0162 and it should be part of 2.2.

**Comment by xephon2** [comment]
The rule is fixed and therefore an error is thrown here, because this `<param>` element uses the attribute `enum`. Please have a look at [line 35 of the plugin.xml of org.dita.pdf2](https://github.com/dita-ot/dita-ot/blob/develop/src/main/plugins/org.dita.pdf2/plugin.xml#L35). There you can see, that the element is empty.

**Comment by jelovirt** [comment]
I can't reproduce this with `hotfix/2.2.3` code and it should work the same with 2.2.2. The `enum` type allows zero or more `val` elements.

**Comment by xephon2** [comment]
You're right, it was fixed with a32d667. I have overseen that Oxygen uses "its own" `plugin.rnc` (I do not understand for what reason) and does not use the one shipped with the DITA-OT.

`/opt/Oxygen XML Editor 17/frameworks/dita/plugin/plugin.rnc`

Sorry :disappointed:

**Comment by raducoravu** [comment]
@xephon2 I'll see if I can change the catalog mappings a bit so that Oxygen uses directly the "plugin.rnc" shipped with DITA OT 2.x.
@jelovirt Maybe in the "catalog-dita.xml" we could have an URI mapping like this:

       <uri name="plugin:plugin.rnc" uri="resources/plugin.rnc"/>

which could be used by Oxygen when the plugin.xml has an xml-model PI associating the schema to it:

    <?xml-model href="plugin:plugin.rnc" type="application/relax-ng-compact-syntax"?>
